src/main/java/in/projecteka/consentmanager/link/link/LinkController.java

```diff
@@ -49,4 +53,28 @@
     public Mono<PatientLinksResponse> getLinkedCareContextInternal(@PathVariable String username) {
         return link.getLinkedCareContexts(username);
     }
+
+    /**
+     * This API is intended for a CM App.
+     * e.g. a mobile app or from other channels
+     * @param linkRefNumber
+     * @param patientLinkRequest
+     * @return
+     */
+    @PostMapping("/v1/links/link/confirm")
```

Comment:
If we add comments, better explain what's this API all about too.

Reply:
I plan to crosslink with @link to the Specs.